resources/prep/conda.md

```diff
@@ -90,7 +90,7 @@ The [package](https://github.com/Anaconda-Platform/nb_conda_kernels) that detect
 
 ## Conda on your own computer
 
-Conda may be used on your computer as well as the Hub. If you wish to install the same environment as the hub is running, after you install Conda, you can download the [`environment.yml`](https://github.com/oceanhackweek/ohw20-tutorials/blob/master/environment.yml) that we use, then `conda create -n <ENV NAME> --file environment.yml`
+Conda may be used on your computer as well as the Hub. If you wish to install the same environment as the hub is running, after you install Conda, you can download the [`environment.yml`](https://github.com/oceanhackweek/jupyter-image/blob/main/py-base/environment.yml) that we use, then `conda env create -n <ENV NAME> --file environment.yml`
```

Comment:
This env file is Linux only. Let's keep the reference to the ohw20-tutorials and I'll add a proper env file there. We still need to change `master` to `main`.

Reply:
Sounds good. The link was broken, but there may be a better link to use here, as you say. Thanks, Filipe!

Reply:
We may not have a great option here, for now. The OHW20 env file includes both Python and R, which we may not want keep here. But to at least fix the broken link, here's the correct, updated link:
https://github.com/oceanhackweek/ohw-tutorials/blob/OHW20/environment.yml

Reply:
Thanks @emiliom, I updated the link!

Reply:
Just added a "canonical one" here: https://github.com/oceanhackweek/ohw-tutorials/pull/100

It should be in main so we can port from year to year. Also, I'm pretty sure that one from OHW20 is also platform specific. The one I added above should be platform agnostic, even on the new mac M1/2.